src/absil/ilwritepdb.fs

```diff
@@ -371,34 +371,48 @@ let generatePortablePdb (embedAllSource: bool) (embedSourceList: string list) (s
                     else
                         // Sequence-point-record
                         let offsetDelta =
-                            if i > 0 then sps.[i].Offset - sps.[i - 1].Offset                           // delta from previous offset
-                            else sps.[i].Offset                                                         // delta IL offset
+                            let offset = sps.[i].Offset
+                            if offset > 0x20000000 then 0                                               //offset out of range
+                            elif i > 0 then offset - sps.[i - 1].Offset                                 // delta from previous offset
+                            else offset                                                                 // delta IL offset
 
                         if i < 1 || offsetDelta <> 0 then                                               // ILOffset must not be 0
                             builder.WriteCompressedInteger(offsetDelta)
 
-                            if sps.[i].Line = 0xfeefee && sps.[i].EndLine = 0xfeefee then               // Hidden-sequence-point-record
+                            let capLine v = if v < 0 || v >= 0x20000000 then 0xfeefee else v
```

Comment:
The use of `0xfeefee` here is very odd.  FEEFEE has a special meaning for a hidden sequence point https://blogs.msdn.microsoft.com/jmstall/2005/06/19/line-hidden-and-0xfeefee-sequence-points/.  I think this should be something else.

Reply:
Since the data is invalid, I can throw the sequence point away instead of writing a hidden sequence point.

Lets see what that looks like.